api/connector.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable no-param-reassign */
+const setupSocket = (socket, game, player, onSubscribe) => {
```

Comment:
remove onSubscribe

* difficult to understand what does it mean ?
* starting a game should be under organizer responsability

=> add a msg READY

-> JOIN
JOINED <-
-> READY
START <-